src/pages/iou/steps/MoneyRequestAmountPage.js

```diff
@@ -233,6 +237,16 @@ class MoneyRequestAmountPage extends React.Component {
         return title[this.iouType];
     }
 
+    /**
+     * Check and dismiss modal
+     */
+    shouldHideModal() {
```

Comment:
```suggestion
    dismissModalIfNecessary() {
```

We could name this function better here